### PR TITLE
Bimodal MCA unit test

### PR DESCRIPTION
**Pull request by SagiPolaczek**
# Ready for CR 🎉 

### GOAL
Test the Bimodal MCA runner.

### Changes
Added two unit test:
1. `test_data_and_model()` - Loads the data and performs one forward pass as a sanity check for the model
2. `test_runner()` - Runs one epoch **with a subset of 1K sample ids**.  - Note that I had to support it in the data module itself.

Beside that I did some typing fixes to the files I touched.

### CODE REVIEW

File: fusedrug_examples/interaction/drug_target/affinity_prediction/bimodal_mca/runner.py [hunk 1]
```diff
@@ -4,11 +4,13 @@
...
 import os
 from omegaconf import DictConfig, OmegaConf
+from fusedrug_examples.interaction.drug_target.affinity_prediction.bimodal_mca import model, data, utils
 import hydra
 import socket
-import model
-import data
-import utils
+
+# import model
```
Comment (alex-golts) [inline fusedrug_examples/interaction/drug_target/affinity_prediction/bimodal_mca/runner.py:11]: you can remove the comments too :)

**Review by alex-golts — APPROVED** [review]
Thanks Sagi, looks great!
I hope it was easy enough to implement the test

**Comment by SagiPolaczek** [comment]
> Thanks Sagi, looks great! I hope it was easy enough to implement the test

Thanks! And thanks for the quick review!!

I think I might have some more fixes to do plus I wanna merge it after the CICD PR 🚀 
I'll re-request a CR when finished if that's OK :)

**Review by mosheraboh — APPROVED** [review]
Looks great

**Pull request merged**